actions/common.go

```diff
@@ -1,20 +1,43 @@
 package actions
 
 var (
-	repoNames = []string{
-		"builder",
-		"controller",
-		"dockerbuilder",
-		"fluentd",
-		"logger",
-		"minio",
-		"postgres",
-		"registry",
-		"router",
-		"slugbuilder",
-		"slugrunner",
-		"workflow",
-		"workflow-e2e",
-		"workflow-manager",
+	// TODO: pull repo list from canonical source -- perhaps workflow-manager-api?
```

Comment:
might be good to create an issue for this TODO in this repo. not blocking for this PR of course